added content on dashboard alerts (#6156)
open-cluster-management/rhacm-docs: observability/design_grafana.adoc

[#using-grafana-dashboards]
= Using Grafana dashboards

> Use Grafana dashboards to view hub cluster and managed cluster metrics. The data displayed in the Grafana alerts dashboard relies on `alerts` metrics, originating from managed clusters. The `alerts` metric has nothing to do with managed clusters forwarding alerts to {prod-short} alert manager on the hub cluster. Therefore, the metrics and alerts have distinct propagation mechanisms and follow separate code paths. 

Reviewer: The correct conref for the product is `{product-title-short}`. The conrefs are listed in any of the `common-attributes.adoc` files in the rhacm-docs repo.

```suggestion
Use Grafana dashboards to view hub cluster and managed cluster metrics. The data displayed in the Grafana alerts dashboard relies on `alerts` metrics, originating from managed clusters. The `alerts` metric does not affect managed clusters forwarding alerts to {product-title-short} alert manager on the hub cluster. Therefore, the metrics and alerts have distinct propagation mechanisms and follow separate code paths. 
```

> Even if you see data in the Grafana alerts dashboard, that does not guarantee that the managed cluster alerts are successfully forwarding to the {prod-short} hub cluster alert manager. If the metrics are propagated from the managed clusters, you can view the data displayed in the Grafana alerts dashboard. 

Reviewer: ```suggestion
Even if you see data in the Grafana alerts dashboard, that does not guarantee that the managed cluster alerts are successfully forwarding to the {product-title-short} hub cluster alert manager. If the metrics are propagated from the managed clusters, you can view the data displayed in the Grafana alerts dashboard. 
```
